projects/1.16.1/assets/cazfps-the-dead-sea/cazfps_the_dead_sea/lang/zh_cn.json

```
@@ -0,0 +1,93 @@
+  "block.cazfps_the_dead_sea.steelwall": "钢墙",
+  "item.cazfps_the_dead_sea.deadclown": "死亡小丑 刷怪蛋",
+  "item.cazfps_the_dead_sea.oldman_spawn_egg": "可怖老人 刷怪蛋",
+  "block.cazfps_the_dead_sea.deadstonestairs": "死之石楼梯",
+  "advancements.tds.title": "死亡之海",
+  "entity.cazfps_the_dead_sea.deadwalker": "死亡游荡者",
+  "block.cazfps_the_dead_sea.deaddirt": "死亡土",
+  "item.cazfps_the_dead_sea.bigfoot_spawn_egg": "大脚巨人 刷怪蛋",
+  "block.cazfps_the_dead_sea.pathblock": "石路",
+  "key.mcreator.tdsk": "死亡之海 菜单",
+  "block.cazfps_the_dead_sea.deadgrass": "死亡草",
+  "item.cazfps_the_dead_sea.crosshearth": "Cross hearth amo",
+  "block.cazfps_the_dead_sea.deadtreeplank": "死亡之树木板",
+  "entity.cazfps_the_dead_sea.deadwitch": "死亡女巫",
+  "block.cazfps_the_dead_sea.steelslab": "钢台阶",
+  "block.cazfps_the_dead_sea.dsslab": "死亡石台阶",
+  "block.cazfps_the_dead_sea.deadsand": "死亡沙",
+  "block.cazfps_the_dead_sea.dps": "死亡之树木板楼梯",
+  "block.cazfps_the_dead_sea.deadtreebrench": "死亡之树树枝",
+  "block.cazfps_the_dead_sea.annimateddeadtreelog": "死亡之树原木（活）",
+  "item.cazfps_the_dead_sea.deadclown_spawn_egg": "死亡小丑 刷怪蛋",
+  "block.cazfps_the_dead_sea.dtps": "死亡之树木台阶",
+  "block.cazfps_the_dead_sea.deadtreeslab": "死亡之树台阶",
+  "block.cazfps_the_dead_sea.handplant": "死之草",
+  "item.cazfps_the_dead_sea.oldman": "可怖老人 刷怪蛋",
+  "entity.cazfps_the_dead_sea.sirenhead": "塞壬之头",
+  "advancements.tff.title": "遗忘之森",
+  "block.cazfps_the_dead_sea.concretestairs": "混凝土楼梯",
+  "block.cazfps_the_dead_sea.steelstairs": "钢楼梯",
+  "block.cazfps_the_dead_sea.deadbush": "大型死之草丛",
+  "block.cazfps_the_dead_sea.deadstonebrick": "死之石砖",
+  "entity.cazfps_the_dead_sea.oldman": "可怖老人",
+  "item.cazfps_the_dead_sea.sirenhead_spawn_egg": "塞壬之头 刷怪蛋",
+  "block.cazfps_the_dead_sea.deadsandlayerbigg": "大层死之沙",
+  "item.cazfps_the_dead_sea.gunamo": "子弹",
+  "entity.cazfps_the_dead_sea.deadclown": "死亡小丑",
+  "advancements.deadclownw.descr": "gg!",
+  "block.cazfps_the_dead_sea.stair": "石路楼梯",
+  "block.cazfps_the_dead_sea.dsbslab": "死之石砖台阶",
+  "block.cazfps_the_dead_sea.concrete": "混凝土",
+  "block.cazfps_the_dead_sea.steelfence": "钢栅栏",
+  "block.cazfps_the_dead_sea.dtpw": "死之木板树墙",
+  "block.cazfps_the_dead_sea.deadstone": "死之石",
+  "block.cazfps_the_dead_sea.deadstonebrickwall": "死之石砖墙",
+  "block.cazfps_the_dead_sea.hangingbody": "吊尸",
+  "block.cazfps_the_dead_sea.concreteslab": "混凝土台阶",
+  "advancements.soa.title": "灰烬之街",
+  "block.cazfps_the_dead_sea.slabb": "石路台阶",
+  "block.cazfps_the_dead_sea.deadsandlayer": "死之沙层",
+  "block.cazfps_the_dead_sea.deadstonebrickstairs": "死之石砖楼梯",
+  "block.cazfps_the_dead_sea.back": "回到主世界",
+  "advancements.soa.descr": "欢迎！！",
+  "block.cazfps_the_dead_sea.deadstonewall": "死之石墙",
+  "block.cazfps_the_dead_sea.cross_2": "十字架之尸下部分",
+  "item.cazfps_the_dead_sea.deadwitch": "死亡女巫 刷怪蛋",
+  "item.cazfps_the_dead_sea.deadwitch_spawn_egg": "死亡女巫 刷怪蛋",
+  "advancements.tdg.title": "死亡花园",
+  "block.cazfps_the_dead_sea.deadstonefence": "死之石栅栏",
```

统一一下风格可能更好（死亡之海、死亡之树、死亡之… 一以贯之）
```suggestion
  "block.cazfps_the_dead_sea.deadstonefence": "死亡之石栅栏",
```